src/components/Discussion/Internal/Comment/Header.js

```diff
@@ -294,59 +269,39 @@ export const Header = ({ t, comment, menu, isExpanded, onToggle }) => {
           )}
         </div>
       </div>
-      {onToggle && (
-        <button
-          {...headerActionStyle({ isExpanded })}
-          {...headerActionStyleHover}
-          {...colorScheme.set('color', isExpanded ? 'divider' : 'textSoft')}
-          onClick={onToggle}
-        >
-          {!isExpanded && comments && comments.totalCount > 0 && (
-            <div {...styles.expandCount}>
-              {t.pluralize('styleguide/comment/header/expandCount', {
+      <div {...styles.actionsWrapper} className={styles.actionsWrapper}>
+        {onToggle && (
+          <IconButton
+            invert={true}
+            Icon={isExpanded ? UnfoldLessIcon : UnfoldMoreIcon}
+            onClick={onToggle}
+            label={
+              !isExpanded &&
+              t.pluralize('styleguide/comment/header/expandCount', {
```

Comment:
Current behavior is not to show the expandCount label on mobile, I would keep it that way.